Assets/Scripts/Weapons/LookAtMouse.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) TigardHighGDC
+// SPDX-License SPDX-License-Identifier: Apache-2.0
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class LookAtMouse : MonoBehaviour
+{
+    public Camera Camera;
+
+    private void Update()
+    {
+        var mousePosition = Camera.ScreenToWorldPoint(Input.mousePosition);
+        var relativePoint = transform.position - mousePosition;
+        transform.rotation = Quaternion.Euler(0, 0, Mathf.Atan2(relativePoint.y, relativePoint.x) * Mathf.Rad2Deg + 90);

```

Comment:
This is the only time so far we are using `var`. Any particular reason we are doing this here?

Edit: I am now seeing that `var` is also used in `Gun.cs`. It's not my favorite but if there is a good reason for it I don't mind using it.

Reply:
There isn't a strong reason to `var`. It was simply used to make it easier to program at the time since it used many different types.

Reply:
Should we move away from it then? I prefer to have things be as explicit as possible

Reply:
This doesn't need to be left in. It was only used to programming it easier at the moment. I forgot to make the changes.

Reply:
Ok I'll push a commit that changes `var` to it's container data type.